Kickstarter-iOS/Features/PledgePaymentMethods/Controller/PledgePaymentMethodsViewController.swift

```diff
@@ -139,6 +140,14 @@ final class PledgePaymentMethodsViewController: UIViewController {
 
         strongSelf.updateAddNewPaymentMethodButtonLoading(state: showLoadingIndicator)
       }
+
+    self.viewModel.outputs.notifyFacebookCAPIUserEmail
+      .observeForUI()
```

Comment:
Oops missed this one, no need for observe for UI, can go straight to `observeValues`